data.yml

```diff
@@ -207,6 +207,16 @@ landscape:
             twitter: https://twitter.com/AirtrainAI
             crunchbase: https://www.crunchbase.com/organization/sematic-2bbe
             logo: airtrain.svg
+      - subcategory:
+            name: Generative AI Platform
```

Comment:
The term "Generative AI Platform" is too broad for my understanding. Could you please provide a more specific context or description?

Reply:
we are an end to end platform company like Google Gen AI studio and fall under multiple sub categories. what would be the right term in this case?

Reply:
How about end-to-end platform?

Reply:
yes, this would work

Reply:
Should I create a new PR, with end-to-end subcategory?